components/table/test/table.vdiff.js

```diff
@@ -0,0 +1,1229 @@
+import '../../inputs/input-number.js';
+import '../../inputs/input-text.js';
+import '../demo/table-test.js';
+import '../table-col-sort-button.js';
+import { defineCE, expect, fixture, focusElem, html, nextFrame } from '@brightspace-ui/testing';
+import { LitElement } from 'lit';
+import { tableStyles } from '../table-wrapper.js';
+
+const colSortButtonFixture = html`
+	<table class="d2l-table">
+		<thead>
+			<tr>
+				<th><d2l-table-col-sort-button>Ascending</d2l-table-col-sort-button></th>
+				<th><d2l-table-col-sort-button desc>Descending</d2l-table-col-sort-button></th>
+				<th><d2l-table-col-sort-button nosort>No Sort</d2l-table-col-sort-button></th>
+			</tr>
+		</thead>
+		<tbody>
+			<tr>
+				<td>Cell 1-A</td>
+				<td>Cell 1-B</td>
+				<td>Cell 1-C</td>
+			</tr>
+		</tbody>
+	</table>
+`;
+
+describe('d2l-table', () => {
+
+	['ltr', 'rtl'].forEach((dir) => {
+		describe(dir, () => {
+			['default', 'light'].forEach((type) => {
+
+				async function createFixture(markup, opts) {
+					const rtl = (dir === 'rtl');
+					const tag = defineCE(
+						class extends LitElement {
+							static get styles() {
+								return [tableStyles];
```

Comment:
Table styles need to be applied from a Lit component, which is why there were so many test components before. Now we can just use `defineCE`, but it means these all need to go through this wrapper.